Library/Formula/libxml2.rb

```diff
@@ -28,12 +18,7 @@ class Libxml2 < Formula
 
   option :universal
 
-  depends_on :ld64
-
-  fails_with :llvm do
-    build 2326
-    cause "Undefined symbols when linking"
-  end
+  depends_on "xz" => :build
```

Comment:
This isn't a build dep; it links dynamically against it, so it'll also be needed at runtime.